zplugin-dds/src/lib.rs

```diff
@@ -759,7 +759,7 @@ impl<'a> DdsPluginRuntime<'a> {
                             }
                             // Workaround for the DDS Writer to correctly match with a FastRTPS Reader declaring a Reliability max_blocking_time < infinite
                             if qos.reliability.max_blocking_time < DDS_INFINITE_TIME {
-                                qos.reliability.max_blocking_time += 1;
+                                qos.reliability.max_blocking_time = qos.reliability.max_blocking_time.saturating_add(1);
                             }
```

Comment:
`DDS_INFINITE_TIME` is equal to `i64::MAX`. So, there is no need for the `if` check. Therefore, the whole `if` block should be simply replaced with:
```
qos.reliability.max_blocking_time = qos.reliability.max_blocking_time.saturating_add(1);
```